.circleci/config.yml

```diff
@@ -36,6 +36,7 @@ jobs:
     steps:
       - checkout
       - run: yarn install --non-interactive
+      - run: yarn danger ci --failOnErrors --id @brainhubeu/react-carousel
```

Comment:
perhaps it would be a better idea to run this as postinstall script?

Reply:
this command works only in CI